storage/oasis/runtime.go

```diff
@@ -56,7 +56,7 @@ func (rc *RuntimeClient) nativeTokenSymbol() string {
 		// so conflicts (particularly intentional/malicious) are possible.
 		// https://github.com/oasisprotocol/oasis-indexer/pull/362#discussion_r1153606360
 		for _, paratime := range network.ParaTimes.All {
-			if paratime.ID == rc.info.ID.Hex() {
+			if paratime.ID == rc.sdkPT.ID {
```

Comment:
this is a minimal change to keep things compiling. should just use rc.sdkPT instead of searching in DefaultNetworks in a follow up

Reply:
it's a trivial change though, right? This whole method just becomes `return rc.sdkPT.Denominations[sdkConfig.NativeDenominationKey].Symbol`?
If so, let's just do it here.

Reply:
k